src/apps/intel/intel_app.lua

```diff
@@ -35,7 +35,7 @@ end
 -- Create an Intel82599 App for the device with 'pciaddress'.
 function Intel82599:new (arg)
    local conf = config.parse_app_arg(arg)
-
+   local result
```

Comment:
this change seems unnecessary or an artifact of previous things; please remove:)